sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -42,11 +42,21 @@ class DatalabAppElement extends Polymer.Element {
    */
   public routeData: Object;
 
+  private _boundResizeHandler: EventListenerObject;
+
   constructor() {
     super();
 
     // Set the pattern once to be the current document pathname.
     this.rootPattern = (new URL(this.rootPath)).pathname;
+
+    this._boundResizeHandler = this._resizeHandler.bind(this);
+    window.addEventListener('resize', this._boundResizeHandler, true);
+
+    // Will be called after the custom element is done rendering.
+    window.addEventListener('WebComponentsReady', () => {
```

Comment:
Is there a reason this call and the one on line 54 use different approaches to calling the same function?
Alternatively, can you call _resizeHandler from ready() instead of using a listener?